This macro conflicts with the select syscall. Since it isn't actually used, we
just remove it.
--- opencl/source/builtin_kernels_simulation/opencl_c.h.orig	2020-04-30 12:24:42.000000000 +0200
+++ opencl/source/builtin_kernels_simulation/opencl_c.h	2020-05-12 22:31:37.166337603 +0200
@@ -220,8 +220,6 @@
     (                               \
         (type)var)
 
-#define select(a, b, c) (c ? b : a)
-
 uint get_local_id(int dim);
 uint get_global_id(int dim);
 uint get_local_size(int dim);
